sdks/python/README.md

```diff
@@ -2,18 +2,92 @@
 
 The Redactive Python SDK provides a robust and intuitive interface for interacting with the Redactive platform, enabling developers to seamlessly integrate powerful data redaction and anonymization capabilities into their Python applications.
 
-## Building the Python SDK
+## Documentation
 
-`cd sdks/python`
+## Installation
 
-`python -m pip install --upgrade build`
+You don't need this source code unless you want to modify the package. If you just
+want to use the package, just run:
 
-`python -m build`
+```sh
+pip install --upgrade redactive
+```
 
-`python install -e .`
+Install from source with:
 
-## Building the optional reranker
+```sh
+python setup.py install
+```
 
-`cd sdks/python`
+### Requirements
 
-`pip install -e ".[reranking]"`
+- Python 3.11+
+
+## Usage
+
+The library has following components.
+
+- **AuthClient** - provides functionality to interact with data sources
+- **SearchClient** - provides functionality to search chunks with Redactive search service in gRpc
+- **RerankingSearchClient** [Experimental] - provides ranked search
```

Comment:
Retrieves extra results, then re-ranks them using a more precise ranking function, returning the top_k results.